.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+- DO NOT ask questions using GitHub issues (only bug or feature requests please,
+  see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)
+
+- Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or
+  https://gitter.im/strongloop/loopback
+
+- Immediate support is available through our subscription plans, see
+  https://strongloop.com/node-js/subscription-plans
+-->
+
+### Bug or feature request
+
+- [ ] Bug
+- [ ] Feature request
+
+### Description of feature (or steps to reproduce if bug)
```

Comment:
@bajtos Added steps to reproduce in this step and left description to cover the case where it is a feature (and therefore has no STR).